cmd/create/machinepool/machinepool.go

```diff
@@ -554,7 +554,8 @@ func addMachinePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster
 		}
 	} else {
 		r.Reporter.Infof("Machine pool '%s' created successfully on cluster '%s'", name, clusterKey)
-		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool --machinepool %s'", name)
+		r.Reporter.Infof("To view the machine pool details, run 'rosa describe machinepool -c %s --machinepool %s'",
```

Comment:
Would it be better to use the long flag :thinking: `--cluster` 

Probably should look to align this across the CLI

Reply:
updated